examples/gno.land/r/gnoland/pages/page_contribute.gno

```diff
@@ -0,0 +1,93 @@
+
+Participants in the gno.land Bounty Program must meet the legal Terms and Conditions referenced [here](https://docs.google.com/document/d/1aXrZ6japdAykB5FLmHCCeBZTo-2tbZQHSQi79ITaTK0/edit).
+
+### Bounty sizes
+
+Each bounty is associated with a size, to which corresponds the maximum compensation for the work involved on the bounty.
+
+In some cases, the work associated with a bounty may be outstanding. When that happens, the core team can decide to award up to 25% of the bounty's value to the recipient.
+
+The value of the bounty, aside from the material completion of the task, considers the involved time in managing the created pull request and iterating on feedback.
+
+
+t-shirt size | expected compensation
+-------------|-----------------------
+[XS]         | $ 500
+[S]          | $ 1000
+[M]          | $ 2000
+[L]          | $ 4000
+[XL]         | $ 8000
+_[XXL]_ \*   | $ 16000
+_[3XL]_ \*   | $ 32000
+
+[XS]: https://github.com/gnolang/gno/labels/bounty%2FXS
+[S]: https://github.com/gnolang/gno/labels/bounty%2FS
+[M]: https://github.com/gnolang/gno/labels/bounty%2FM
+[L]: https://github.com/gnolang/gno/labels/bounty%2FL
+[XL]: https://github.com/gnolang/gno/labels/bounty%2FXL
+[XXL]: https://github.com/gnolang/gno/labels/bounty%2FXXL
+[3XL]: https://github.com/gnolang/gno/labels/bounty%2F3XL
+
+\*: XXL and 3XL bounties are exceptional. Almost no issues will have these sizes; most will be broken down into smaller bounties.
+
+## gno.land Grants
+
+The gno.land grants program is to encourage and support the growth of the gno.land contributor community, and build out the usability of the platform and smart contract library. The program provides financial resources to contributors to explore the Gno tech stack, and build dApps, tooling, infrastructure, products, and smart contract libraries in gno.land.
+
+TODO: Add link to new repo
+
+## Join Game of Realms
+
+Game of Realms is the overarching contributor network of gnomes, currently running off-chain, and will eventually transition on-chain. At this stage, a Game of Realm's contribution is comprised of high-impact contributions identified as 'notable contributions'.
+
+These contributions are not linked to immediate financial rewards, but are notable in nature, in the sense they are a challenge, make a significant addition to the project, and require persistence, with minimal feedback loops from the core team.
+
+You can always contribute to the project, and all contributions will be noticed. Contributing now is a way to build your personal contributor profile in gno.land early on in the ecosystem, and signal your commitment to the project, the community, and its future.
+
+There are a variety of ways to make your contributions count:
+- Core code contributions
+- Realm development
```

Comment:
Maybe mention packages as well?

Reply:
changed to "realm and pure package development"